lib/puppet/settings.rb

```diff
@@ -807,10 +834,9 @@ def any_files_changed?
   private :any_files_changed?
 
   def reuse
```

Comment:
Not that this is _this_ PR's fault, but I find this method name vague and mildly misleading...